script/remove-unnecessary-raw-entity-files.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+ALL_FILES=$(mktemp)
+READ_FILES=$(mktemp)
+TEST_ENTITIES_PATH="$(dirname "$0")"/../src/site/stages/build/process-cms-exports/tests/entities
+
+# Build a list of all the files (ls)
+ls $TEST_ENTITIES_PATH | sort | uniq > "$ALL_FILES"
```

Comment:
I don't think `uniq` is necessary, since it isn't possible to have two identically named files in the same directory.

Reply:
Oh, yeah. Good call. Habit. :sweat_smile: